source/reference/minio-mc/mc-ilm-rule-edit.rst

```diff
@@ -255,6 +255,8 @@ Parameters
    marks an object as eligible for transition. MinIO transitions the object to
    the configured remote storage tier specified to the 
    :mc-cmd:`~mc ilm rule edit --transition-tier`. 
+   Specify the number of days as an integer, e.g. ``30`` for 30 days.
+   If the remote tier is another MinIO deployment, transition days can be set to ``0`` to mark new objects for the quickest move to the remote tier.
```

Comment:
wording

```suggestion
   If the remote tier is another MinIO deployment, you can set the value to to ``0`` to mark new objects as immediately eligible for transition to the remote tier.
```

Removes passive, and emphasizes eligibility

Reply:
With a minor edit to remove the duplicate "to".